BrownField/Networking/Concepts/Connectivity-Multi-region-with-vWAN/Dual-Region-with-vWAN.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Network considerations for Azure VMware Solution dual-region deployments with Secure vWAN
+description: Learn how to configure network connectivity when Azure VMware Solution private clouds are deployed in two Azure regions with Secure vWAN.
+author: jmedina
+ms.author: jmedina
+ms.date: 06/22/2023
+ms.topic: conceptual
+ms.service: caf
+ms.subservice: caf-scenario-vmware
+ms.custom: think-tank, e2e-azure-vmware
+---
+
+# Network considerations for Azure VMware Solution dual-region deployments using Secure vWAN with Routing-Intent
+
+This article describes the best practices for connectivity, traffic flows, and high availability of dual-region Azure VMWare Solution when using Azure vWAN with Routing Intent. This article will break down vWAN with Routing Intent topology from the perspective of AVS private clouds, On-premise sites, and Azure native. How to implement and configure Secure vWAN with Routing Intent is out of the scope and will not be discussed in this document.
+
+The document assumes readers have a basic understanding of vWAN and Secure vWAN with Routing Intent.
+
+**What is Azure Virtual WAN?**  
+[https://learn.microsoft.com/en-us/azure/virtual-wan/how-to-routing-policies](https://learn.microsoft.com/en-us/azure/virtual-wan/virtual-wan-about)
+
+**How to configure Virtual WAN Hub routing intent and routing policies**  
+https://learn.microsoft.com/en-us/azure/virtual-wan/how-to-routing-policies
+
+## Dual-region with Secure vWAN scenario  
+Secure vWAN with Routing Intent provides the capability to send all Internet traffic and Private network traffic (RFC 1918) to a security solution like Azure Firewall, a third-party NVA, or SaaS. 
+In the scenario, we have a network topology that spans two regions. There is one vWAN with two Hubs, Hub1 and Hub2. Hub1 is in Region 1, and Hub2 is in Region 2.Each Hub in both regions has its instance of Azure Firewall deployed(Hub1Fw, Hub2Fw), essentially making them Secure vWAN Hubs. Having Secure vWAN hubs is a technical prerequisite to Routing Intent. Secure vWAN Hub1 and Hub2 have Routing Intent enabled.  
+
+Each region has its own AVS Private Cloud and an Azure Vnet. There is also an on-premise site connecting to both regions, which we will review in more detail later in this document.
+![image](https://github.com/jasonamedina/Enterprise-Scale-for-AVS/assets/97964083/0d2d0b80-e550-4f69-a321-411658a066ab)
+### Understanding Topology Connectivity 
+**Brown Connections**: AVS private cloud connection to its local regional hub.  
+**Pink Connections**: AVS private cloud connection to its cross-regional hub.   
+**Orange Connection**: AVS Region 1 Global Reach connection back to on-premise.   
+**Green Connection**: AVS Region 2 Global Reach connection back to on-premise.   
+**Purple Connection**: AVS Region 1 and AVS Region 2 Global Reach connection back to each other's private cloud.   
```

Comment:
Something like "Global Reach connection between the two private clouds' managed circuits" might be easier for readers to understand

Reply:
@fguerri, I updated the verbiage for "Purple Connection" to "Azure VMWare Solution Global Reach connection between the two private clouds' managed circuits."